src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
All of the open source projects that I have ever worked on follow PSRs 1 and 2 and it's the style that I code in day-to-day.

There is methodology behind it (use braces on the same line for blocks you flow into) and I think it's the style most of our consumers will be more used to.